### ISSUE TITLE
Web panel bug with mysql connection.

### ISSUE THREAD

**Issue by Vlad0sMiner** [VolmitSoftware/PermissionsEx#19]
Hello, I'm sorry for my bad English, I'm from Russia. 
When connecting via mysql, the web control panel behaves strangely. It doesn't load any records, and if you create anything in it, it will delete absolutely everything in the connected database table. After saving, the created record also disappears. The control panel is completely empty. Everything is normal when working with YAML.

**Comment by AvianAeternum** [comment]
Yep apologies, the web panel does not support database storage. Ive directed the primary use of this fork of pex towards personal use, and or for developers and server admins so they can rapidly configure local instances, or single instances etc.

**Comment by Vlad0sMiner** [comment]
Are you planning to add working with the database via the dashboard?

**Comment by AvianAeternum** [comment]
No, I do not intend to work on this fork further. As zml has begin to update the main fork again.
